src/config.py

```diff
@@ -1,31 +1,93 @@
 import hashlib
 import json
 import urllib.parse
+import os
+
+from language import Language, DEFAULT_LANGUAGE
+
+SUPPORTED_YEARS = []
+DEFAULT_YEAR = '2019'
 
 config_json = {}
 
 DEFAULT_AVATAR_FOLDER_PATH = '/static/images/avatars/'
 AVATAR_SIZE = 200
 AVATARS_NUMBER = 15
 
+SUPPORTED_CHAPTERS = {}
+SUPPORTED_LANGUAGES = {}
+
 def get_config(year):
   global config_json
   return config_json[year] if year in config_json else None
 
+
+def get_entries_from_json(json_config, p_key, s_key):
+  entries = []
+  if p_key in json_config:
+    for values in json_config.get(p_key):
+      entries.append(values.get(s_key))
+  return entries
+
+
+def get_chapters(json_config):
+  chapters = []
+  data = get_entries_from_json(json_config,'outline','chapters')
+  for list in data:
+    for entry in list:
+      chapters.append(entry.get('slug'))
+  return chapters
+
+
+def get_languages(json_config):
+  languages = []
+  data = get_entries_from_json(json_config,'settings','supported_languages')
+  for list in data:
+    for entry in list:
+      languages.append(getattr(Language,entry))
+  return languages
+
+
+def get_live(json_config):
+  isLive = False
```

Comment:
Nit: Python style is to use lowercase and underscores for variable names.
```suggestion
  is_live = False
```

Reply:
Cool. Will fix. First dabble into Python here.